app/javascript/react/screens/App/Settings/__tests__/__snapshots__/SettingsReducer.test.js.snap

```diff
@@ -287,8 +287,10 @@ Object {
   "isSavingSettings": false,
   "postConversionHostsResults": Array [],
   "savedSettings": Object {
+    "cpu_limit_per_host": undefined,
```

Comment:
We should add these fields to the SettingsReducer tests so they appear in the snapshots here.

Reply:
You're right. Fixed.